### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by villesau**
Pull request template helps us to review since it encourages to add links to documentation and npm.

Anything else that would be good to have in PR template?

More details: https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- Links to documentation:
+- Link to NPM:
```
Comment (pascalduez) [inline PULL_REQUEST_TEMPLATE.md]: Maybe we should prioritize Github repo links to npm. I often found myself checking the code to assert some points.
Also sometimes npm packages don't have a repo link.

Comment (villesau) [reply PULL_REQUEST_TEMPLATE.md]: true!

**Comment by villesau** [comment]
Points we could add:

> Type of contribution: new definition | addition | fix | refactor

:+1: 
> Flow version support (yes, should already be in the filenames :)

Hmm yeah why not!

> Reminder about describe, it

It's mentioned in CONTRIBUTING.md already. Should we link that or explicitly mention describe & it?

**Comment by pascalduez** [comment]
> It's mentioned in CONTRIBUTING.md already. Should we link that or explicitly mention describe & it?

Either way but should be present in the template IMO, that seem to be the most common commnent we make :)

Also maybe a suggestion about commit messages, MR  titles.
I find it useful to see the lib name, most often in brackets.  
`[react-redux] Fix all the things`   
`[react-redux_v2.3] Fix a minor thing`

**Comment by villesau** [comment]
I added a comment about it. Feel free to improve later on if something comes to your mind!

**Pull request merged**